KarhooUISDK/Screens/Rides/RidesList/RideCell/RideCellViewModel.swift

```diff
@@ -18,6 +18,7 @@ final class RideCellViewModel {
     public let trip: TripInfo
     public let tripDetailsViewModel: TripDetailsViewModel
     public let showActionButtons: Bool
+    public let isPrebook: Bool
```

Comment:
This property does not need to be exposed as a public property. It can be a local `let` in the init. In the current state it gives the impression that some external entity is using it.

The same for the `isPrebook` property of the `TripMetadataViewModel`.